spec/features/merchants/invoices/show_spec.rb

```diff
@@ -41,7 +41,6 @@
 
     it "total revenue that will be generated from all of items on invoice" do
       visit "/merchants/#{@merchant_1.id}/invoices/#{@invoice_1.id}"
-      save_and_open_page
       expect(page).to have_content("Total Revnue: ")
```

Comment:
You again are going to want to add info to your spec that tests the functionality of the code your passing it rather than what the view is going to output.

Reply:
Also, i see a typo on line 44 that might throw an error